assets/js/blocks/single-product/edit/editor-block-controls.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockControls } from '@wordpress/block-editor';
+import { ToolbarGroup } from '@wordpress/components';
+
+interface EditorBlockControlsProps {
+	isEditing: boolean;
+	setIsEditing: ( isEditing: boolean ) => void;
+}
+
+/**
+ * Adds controls to the editor toolbar.
+ *
+ * @param {Object}                props              Incoming props for the component.
+ * @param {boolean}               props.isEditing
+ * @param {function(boolean):any} props.setIsEditing
+ */
+const EditorBlockControls = ( {
+	isEditing,
+	setIsEditing,
+}: EditorBlockControlsProps ) => {
+	return (
+		<BlockControls>
+			<ToolbarGroup
```

Comment:
For some reason, the toolbar is not visible to me. Does it work for you? :confused: 

![imatge](https://user-images.githubusercontent.com/3616980/226613710-4aa74728-d997-4c77-8253-f4a9966c26f4.png)